src/blocks/ChooseToken.vue

```diff
@@ -19,7 +19,7 @@
         </div>
       </div>
       <div class="tokenListContainer genericListContainer">
-        <div v-for="item in displayedList" :key="item.symbol" class="tokenItem" @click="chooseToken(item)">
+        <div v-for="item in displayedList" :key="item.symbol" class="tokenItem" :data-cy="`token_item_${item.symbol}`" @click="chooseToken(item)">
```

Comment:
Maybe remove `:`? It seems it is not needed here

Reply:
No, seems to be correct since it's a cycle and there is a variable in attribute